app/src/main/res/layout/fragment_amortisationsrechnerpv.xml

```diff
@@ -98,27 +95,27 @@
                     app:layout_constraintTop_toBottomOf="@+id/amortPV_edit_anschaffung" />
 
                 <TextView
-                    android:id="@+id/amortPV_text_vergütung_beschreibung"
+                    android:id="@+id/amortPV_text_ertrag_beschreibung"
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
                     android:text="@string/amortPV_text_ertrag_beschreibung"
                     android:textSize="@dimen/content_text_smaller"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintHorizontal_bias="0.0"
-                    app:layout_constraintStart_toStartOf="@+id/amortPV_edit_kWh_pro_kWp"
-                    app:layout_constraintTop_toBottomOf="@+id/amortPV_text_kWh_pro_kWp" />
+                    app:layout_constraintStart_toStartOf="@+id/amortPV_edit_ertrag"
+                    app:layout_constraintTop_toBottomOf="@+id/amortPV_text_ertrag" />
 
                 <EditText
-                    android:id="@+id/amortPV_edit_kWh_pro_kWp"
+                    android:id="@+id/amortPV_edit_ertrag"
```

Comment:
XML naming and ids should be in snake case only.

Reply:
This will be fixed in the Pull Request from AmortisationsrechnerPV.